src/devices/Tca9548a/Tca9548ADevice.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Device.I2c;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Iot.Device.Tca9548a
+{
+    /// <summary>
+    /// I2C Device on TCA9548A channel
+    /// </summary>
+    public class Tca9548ADevice : I2cDevice
```

Comment:
This is exactly the idea of the I2cDevice, note that unless this offers more than I2cDevice I'd recommend to keep this internal and just return `I2cDevice` from the parent.